gammapy/utils/metadata.py

```diff
@@ -0,0 +1,75 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Metadata base container for Gammapy."""
+import json
+from typing import Optional, Union
+from astropy.coordinates import Angle, EarthLocation, SkyCoord
+from astropy.time import Time
+from astropy.units import Quantity
+import yaml
+from pydantic import BaseModel, validator
+from gammapy.version import version
+
+__all__ = ["MetaData", "CreatorMetaData"]
+
+
+class MetaData(BaseModel):
+    """Base model for all metadata classes in Gammapy."""
+
+    class Config:
+        """Global config for all metadata."""
+
+        arbitrary_types_allowed = True
```

Comment:
Can we change these default values case by case? I can imagine that sometime some metadata are optional and a strict validation might be not wished.